recipes-bsp/grub/grub-bootconf_%.bbappend

```diff
@@ -3,12 +3,13 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/grub:"
 SRC_URI += " \
     file://grub.cfg \
     file://grubenv \
+    file://grub.d \
 "
 
 PACKAGES_prepend = "${PN}-nilrt"
 
-FILES_${PN}-nilrt      += "/boot/efi/nilrt/grub.cfg /boot/grub/grubenv"
-CONFFILES_${PN}-nilrt  += "/boot/efi/nilrt/grub.cfg /boot/grub/grubenv"
+FILES_${PN}-nilrt      += "/boot/efi/nilrt/grub.cfg /boot/grub/grubenv /bootimage.cfg.d/*.cfg"
+CONFFILES_${PN}-nilrt  += "/boot/efi/nilrt/grub.cfg /boot/grub/grubenv /bootimage.cfg.d/*.cfg"
```

Comment:
For my own education - why the separate root folder '/bootimage.cfg.d' as opposed to something under '/boot'?

Reply:
Thats the folder that [grub.cfg](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/recipes-bsp/grub/grub/grub.cfg#L18) is configured to look into. I don't know what the reason was for picking this folder instead of /boot but since that shouldn't affect functionality, I didn't dig deeper.

Reply:
Thanks, that's odd but fine and not worth changing here.